quickstarts/ios-swift/01-login.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Auth0 iOS Swift Quickstarts - 1. Login
+description: This tutorial will show you how to integrate Lock in your iOS Swift project in order to present a login screen.
+name: iOS - Swift
+alias:
+  - ios
+  - iphone
+  - ipad
+  - swift
+  - login
+language:
+  - Swift
+hybrid: false
+image: /media/platforms/ios.png
+tags:
+  - quickstart
+  - login
+alias:
+  - ios
+seo_alias: ios-swift
+---
+
+## iOS Swift - Login Tutorial
+
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your iOS apps with Auth0.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../_includes/_package', {
```

Comment:
We need to update this part accordingly to the new samples and website integration.